operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -175,6 +187,23 @@ The Technical Product Writer/s form part of the PR review/approval process. When
 
 The [Documentation Style guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide) and [UI text guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html).
 
+## Customer Education Processes
+
+### Resolve gaps and painpoints among customers
+
+To resolve gaps and pain points, identify and rank them in priority order.
```

Comment:
Suggestion: Elaborate on the priority order, and possibly provide a goal for why this is being done?

Reply:
like: priority order based on corporate principle *impact* and to reduce load on CSMs and Support Engineers and have more customers use MM with a paid plan?

Reply:
That could work, yep. @jasonblais is best-situated to comment though as he knows more about your specific AORs/goals.

Reply:
```suggestion
To identify, rank and resolve customer pain points. Priority is based on customer impact, measured by an increase in enterprise 90-day onboarding NPS, and post-sales technical onboarding without R&D involvement.
```

Reply:
^the priority is based on the business impact of your [first initiative](https://docs.google.com/document/d/1GpeVX3a1aLnupFIQ63RWVkcNC3enIYmaUXK1gS1HagQ/edit#)